services/app-api/tsconfig.json

```diff
@@ -1,17 +1,19 @@
 {
     "extends": "../../tsconfig.json",
     "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es2021"],
+        "target": "es2021",
+        "moduleResolution": "node",
+
         "baseUrl": "/",
         "experimentalDecorators": true,
         "emitDecoratorMetadata": true,
         "preserveConstEnums": true,
-        "sourceMap": true,
-        "target": "es5",
+        "sourceMap": false,
```

Comment:
Just looking at the scale of these changes to tsconfig for an unrelated effort is like reading the rationale for https://qmacbis.atlassian.net/browse/MR-2672 in code form.